lib/templates.php

```diff
@@ -75,6 +75,17 @@ function gutenberg_register_template_post_type() {
 	);
 
 	register_post_type( 'wp_template', $args );
+
+	$meta_args = array(
+		'object_subtype' => 'wp_part',
```

Comment:
Regardless of the fact I'm not into the meta `theme`, this is a typo! 🙂 
```suggestion
		'object_subtype' => 'wp_template',
```